python/cudf/cudf/tests/test_series.py

```diff
@@ -27,6 +27,8 @@
     gen_rand,
 )
 
+rng = np.random.default_rng(seed=0)
+
 
```

Comment:
*nit*: avoid global rng and make one in the tests?

Reply:
Fixed.